File: Numbers/PrimeNumber.java
public class PrimeNumber{
    static boolean checkPrime(int n){
        int count=0;
        for(int i=1;i<=n;i++){
            if(n%i==0){
                count=count+1;
            }
        }
        if(count==2){
            return true;
        }else{
            return false;
        }

    }
    public static void main(String args[]){
        int a=1, b=20;
        for(int i=a; i<=b;i++){
            if(checkPrime(i))
            System.out.println(i+":is a prime number");
        
        }
    }
}

File: Problem Solving Python/Numbers/Prime_Number.py
def checkPrime():
    num=int(input("Enter a number to check prime or not: "))
    count=0
    for i in range(1,num+1):
        if(num%i==0):
            count+=1
    if(count==2):
        print(f"The number {num} is a prime number")
    else:
        print(f"The number {num} is not a Prime number")

checkPrime()